lib/core/form/components/widgets/date/date.widget.ts

```diff
@@ -57,6 +58,7 @@ export class DateWidgetComponent extends WidgetComponent implements OnInit {
 
         const momentDateAdapter = <MomentDateAdapter> this.dateAdapter;
         momentDateAdapter.overrideDisplayFormat = this.field.dateDisplayFormat;
+        this.field.form.fieldValidators.push(new DateFieldValidator());
```

Comment:
Might be polluting the field instance with validators on each Init event, I suggest moving the validator here: https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/core/form/components/widgets/core/form-field-validator.ts#L528